mlir/test/Dialect/Vector/linearize.mlir

```diff
@@ -318,3 +318,68 @@ func.func @test_vector_extract_scalar() {
   %0 = vector.extract %cst[0] : i32 from vector<4xi32>
   return
 }
+
+// -----
+
+// ALL-LABEL: test_vector_bitcast
+// ALL-SAME: %[[ORIG_ARG:.*]]: vector<4x4xf32>
```

Comment:
[nit] Since the name of the argument is `arg0`.
```suggestion
// ALL-SAME: %[[ARG_0:.*]]: vector<4x4xf32>
```

Reply:
Yes, fixed and thanks.